database.sql

```diff
@@ -1310,6 +1310,7 @@ CREATE TABLE IF NOT EXISTS `post-engagement` (
 	`media-type` tinyint NOT NULL DEFAULT 0 COMMENT 'Type of media in a bit array (1 = image, 2 = video, 4 = audio',
 	`language` varbinary(128) COMMENT 'Language information about this post',
 	`created` datetime COMMENT '',
+	`restricted` boolean NOT NULL DEFAULT '0' COMMENT 'If true, this post is either unlisted or not from a federated network',
```

Comment:
Why not name this field "unlisted"?

Reply:
"unlisted" is a very specific term that is used for public posts that aren't that public. But this field also covers private posts and public posts from connector networks and rss/atom feeds.

Reply:
Hmmm.